ui/app/AppLayouts/Onboarding2/pages/LoginScreen.qml

```diff
@@ -66,14 +70,14 @@ OnboardingPage {
             if (password.length === 0)
                 return
 
-            root.loginRequested(d.settings.lastKeyUid, Onboarding.LoginMethod.Password, {"password": password})
+            root.loginRequested(d.settings.lastKeyUid, Onboarding.LoginMethod.Password, { password })
```

Comment:
That's not the same :) Pls revert

Reply:
But how?
`{ "password": password }`
`{ password: password }`
and shortest `{ password }`

are equivalent. Am I missing something?